src/utils/openmc/tallies.py

```diff
@@ -0,0 +1,1239 @@
+                  'integer'.format(num_realizations)
+            raise ValueError(msg)
+
+        elif num_realizations < 0:
+            msg = 'Unable to set the number of realizations to {0} for ' \
+                  'Tally ID={1} since it is a negative ' \
+                  'value'.format(num_realizations)
+            raise ValueError(msg)
+
+        self._num_realizations = num_realizations
+
+
+    def set_results(self, sum, sum_sq):
+
+        if not isinstance(sum, (tuple, list, np.ndarray)):
+            msg = 'Unable to set the sum to {0}for Tally ID={1} since ' \
+                  'it is not a Python tuple/list or NumPy ' \
+                  'array'.format(sum, self._id)
+            raise ValueError(msg)
+
+        if not isinstance(sum_sq, (tuple, list, np.ndarray)):
+            msg = 'Unable to set the sum to {0}for Tally ID={1} since ' \
+                  'it is not a Python tuple/list or NumPy ' \
+                  'array'.format(sum_sq, self._id)
+            raise ValueError(msg)
+
+        self._sum = sum
+        self._sum_sq = sum_sq
+
+
+    def remove_score(self, score):
+
+        if not score in self._scores:
+            msg = 'Unable to remove score {0} from Tally ID={1} since the ' \
+                  'Tally does not contain this score'.format(score, self._id)
+            ValueError(msg)
+
+        self._scores.remove(score)
+
+
+    def remove_filter(self, filter):
+
+        if not filter in self._filters:
+            msg = 'Unable to remove filter {0} from Tally ID={1} since the ' \
+                  'Tally does not contain this filter'.format(filter, self._id)
+            ValueError(msg)
+
+        self._filters.remove(filter)
+
+
+    def remove_nuclide(self, nuclide):
+
+        if not nuclide in self._nuclides:
+            msg = 'Unable to remove nuclide {0} from Tally ID={1} since the ' \
+                  'Tally does not contain this nuclide'.format(nuclide, self._id)
+            ValueError(msg)
+
+        self._nuclides.remove(nuclide)
+
+
+    def compute_std_dev(self, t_value=1.0):
+
+        # Calculate sample mean and standard deviation
+        self._mean = self._sum / self._num_realizations
+        self._std_dev = np.sqrt((self._sum_sq / self._num_realizations - \
+                                 self._mean**2) / (self._num_realizations - 1))
+        self._std_dev *= t_value
+
+
+    def __repr__(self):
+
+        string = 'Tally\n'
+        string += '{0: <16}{1}{2}\n'.format('\tID', '=\t', self._id)
+        string += '{0: <16}{1}{2}\n'.format('\tName', '=\t', self._label)
+
+        string += '{0: <16}\n'.format('\tFilters')
+
+        for filter in self._filters:
+            string += '{0: <16}\t\t{1}\t{2}\n'.format('', filter._type,
+                                                          filter._bins)
+
+        string += '{0: <16}{1}'.format('\tNuclides', '=\t')
+
+        for nuclide in self._nuclides:
+            string += '{0} '.format(nuclide._name)
```

Comment:
Yes, thanks for bringing this up. This method fails when one of the nuclides is the string "total" rather than an actual ``openmc.Nuclide`` object. I just pushed a fix for this.